src/__tests__/Carousel.js

```diff
@@ -139,22 +139,37 @@ describe("Slider", function() {
 		it("Should disable the left arrow if we are showing the first item", function () {
 			TestUtils.Simulate.click(componentInstance['item0']);
 			expect(ReactDOM.findDOMNode(componentInstance).querySelectorAll('.carousel-slider .control-prev.control-disabled').length).toBe(1);
-		});	
+		});
 
 		it("Should enable the left arrow if we are showing other than the first item", function () {
 			TestUtils.Simulate.click(componentInstance['item1']);
 			expect(ReactDOM.findDOMNode(componentInstance).querySelectorAll('.carousel-slider .control-prev.control-disabled').length).toBe(0);
-		});	
+		});
 
 		it("Should disable the right arrow if we reach the lastPosition", function () {
 			TestUtils.Simulate.click(componentInstance['item1']);
 			expect(ReactDOM.findDOMNode(componentInstance).querySelectorAll('.carousel-slider .control-next.control-disabled').length).toBe(0);
-			
+
 			TestUtils.Simulate.click(componentInstance['item6']);
 			expect(ReactDOM.findDOMNode(componentInstance).querySelectorAll('.carousel-slider .control-next.control-disabled').length).toBe(1);
-		});		
-	})	
+		});
+
+		// describe("Infinite Loop", function () {
+		// 	beforeEach(function () {
+		// 		componentInstance.infiniteLoop = true;
```

Comment:
`componentInstance.setProps({infiniteLoop: true});` should work here.

Soon I will get rid of these poor and old test in favour of brand new using mocha + chai + enzyme